.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,64 @@
-**Goals (and why)**:
+## General
+**Before this PR**:
+
+**After this PR**:
 <!-- User-facing outcomes this PR delivers go below -->
 ==COMMIT_MSG==
 ==COMMIT_MSG==
 
-**Implementation Description (bullets)**:
+**Priority**:
+
+**Concerns / possible downsides (what feedback would you like?)**:
+
+**Is documentation needed?**:
+
+## Compatibility
+**Does this PR create any API breaks (e.g. at the Java or HTTP layers) - if so, do we have compatibility?**:
+
+**Does this PR change the persisted format of any data - if so, do we have forward and backward compatibility?**:
+
+**The code in this PR may be part of a blue-green deploy. Can upgrades from previous versions safely coexist?**:
+
+**Does this PR rely on statements being true about other products at a deployment - if so, do we have correct product dependencies on these products (or other ways of verifying that these statements are true)?**:
+
+**Does this PR need a schema migration?**
+
+## Testing and Correctness
```

Comment:
Perhaps more broadly: What, if any, assumptions are made about the current SOTW? If they change over time, how will we find out? (In my mind, this is different to the scale question)